src/messages/gateway.rs

```diff
@@ -95,7 +97,7 @@ impl Serialize for TaskState {
             TaskState::Invalid => "Invalid",
             TaskState::Waiting => "Waiting",
             TaskState::Finished => "Finished",
-            TaskState::Running => "Running",
+            TaskState::Running(_) => "Running",
```

Comment:
@Kobzol  I wonder if there are implications that the serialization is missing the information about the worker_id

Reply:
In theory, we might need to change the serialization from a string to an enum, but worker placement might be much more complex than a single integer, so maybe this information should be propagated in another way altogether.